<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessagesBrowseClass</class>
 <widget class="QWidget" name="MessagesBrowseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>439</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Message浏览器</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_self">
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="lineEdit_qrpassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_qrlogin">
            <property name="text">
             <string>二维码登录</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit_tongxunlupassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_updateTongXunLu">
            <property name="text">
             <string>更新通讯录</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>群聊：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_group">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>好友：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_friend">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>聊天回朔（h）：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pretime">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>5.0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_autorefresh">
          <property name="text">
           <string>自动刷新（s）：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_autorefresh">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_refresh">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MessagesBrowse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_qrlogin</sender>
   <signal>clicked()</signal>
   <receiver>MessagesBrowseClass</receiver>
   <slot>QR_Login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_updateTongXunLu</sender>
   <signal>clicked()</signal>
   <receiver>MessagesBrowseClass</receiver>
   <slot>GetChatrooms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_autorefresh</sender>
   <signal>clicked(bool)</signal>
   <receiver>MessagesBrowseClass</receiver>
   <slot>AutoReFlush(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MessagesBrowseClass</receiver>
   <slot>ReFlush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>773</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>QR_Login()</slot>
  <slot>GetChatrooms()</slot>
  <slot>AutoReFlush(bool)</slot>
  <slot>ReFlush()</slot>
 </slots>
</ui>
